client/app/account/billing/components/renewDate/billing-renew-date.component.js

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Do we have a specification which recommend this syntax ?

Reply:
https://github.com/johnpapa/angular-styleguide/blob/master/a1/README.md